app/src/main/kotlin/net/sourceforge/opencamera/uploadservice/FileUploadService.kt

```diff
@@ -4,23 +4,35 @@ import android.app.NotificationChannel
 import android.app.NotificationManager
 import android.content.Context
 import android.os.Build
+import androidx.annotation.RequiresApi
+import net.gotev.uploadservice.UploadServiceConfig.initialize
 import net.gotev.uploadservice.protocols.multipart.MultipartUploadRequest
+import net.sourceforge.opencamera.BuildConfig
 import net.sourceforge.opencamera.MainActivity
 import java.io.File
 
 class FileUploadService(private val mMainActivity: MainActivity) {
```

Comment:
Use context - not activity please! Because it will be better to use appcontext here)